modules/collection/programs/fish.nix

```
@@ -198,7 +198,7 @@ in {
           ${concatMapAttrsStringSep "\n" (name: value: "alias -- ${name} ${escapeShellArg (toString value)}") cfg.aliases}
         '';
       }
-      // (mapAttrs' (name: val: nameValuePair ".config/fish/functions/${name}.fish" {source = toFishFunc val "${name}.fish";}) cfg.functions)
+      // (mapAttrs' (name: val: nameValuePair ".config/fish/functions/${name}" {source = toFishFunc val "${name}";}) cfg.functions)
```

Could remove the interpolation too, no?

```suggestion
      // (mapAttrs' (name: val: nameValuePair ".config/fish/functions/${name}" {source = toFishFunc val name;}) cfg.functions)
```